MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+- Any community member can propose the removal of an existing MIP Facilitator or MIP Reviewer by submitting a new MIP.
+- Any community member can vote to approve or reject the selection or removal of MIP Facilitators and MIP Reviewers.
+- As the community builds its capacity to effectively take on these responsibilities, key ecosystem players will continue to fulfill these roles as needed, supporting the bootstrapping and testing of the new processes.
+
+#### Bootstrapping the Process
+
+However, due to the limited current capacity we propose that the Mina Foundation, in collaboration with o1 Labs and other members of the Mina ecosystem, will temporarily fill these new roles as needed while the processes are tested, and the capacity and expertise grows in the community.
+
+This bootstrapping period would last no longer than 12 months.
+
+### Introducing Mass Deliberation Tools and Public Debates
+
+We propose to implement **new tools** to collect and analyze the comments and sentiments from the community at different phases in the MIP process. These include:
+
+- Polling on MinaResearch. The community is encouraged to experiment with different methods to increase effectiveness.
+- Survey bot on Discord. The development and usage of open source AI bots on Discord to gather, organize and summarize mass feedback. The community is encouraged to review and propose improvements to the development:
+https://github.com/MinaFoundation/gptSurveySummarizer 
+- On-Chain Voting. The community is encouraged to suggest any improvements that could be proposed in a future MIP:
+https://github.com/MinaFoundation/mina-on-chain-voting/tree/main 
+https://github.com/MinaFoundation/govbot 
+- Governance Dashboards. The community is encouraged to experiment with different methods and tools to increase participation in the MIP process.
+https://minascan.io/mainnet/governance/MIPs 
+
+We propose to introduce new types of public debates:
+
+- During the expert part of the Review Phase: The MIP Facilitator will schedule, livestream and record open and public meetings between the MIP Author, the MIP Review Group and other stakeholders. The meeting minutes will be posted and shared in the designated channels for public consumption, allowing the community members who are interested to follow the process and contribute with comments. 
+- During the community part of the Review Phase: There will be a period of time for the community to carefully read and consider the MIP and the Review Group’s Risk Assessment. The MIP Facilitator will coordinate with the Mina Foundation to organize a community call where the MIP Author and the MIP Review Group can explain the MIP, including its risks, benefits and tradeoffs, in a way that is comprehensible to non-expert audiences and respond to community questions. Dates for the on-chain vote will be set, along with instructions on how to participate.
```

Comment:
What is the role of foundation here? Why is it needed? Just to make announcements? Or does coordinate imply something more?

Reply:
The Foundation’s role in the community call stage of the Review Phase is primarily supportive and logistical. Specifically, this includes providing logistical and operational support for organizing the calls and leveraging its established communication channels to maximize community awareness and participation.

If the term “coordinate” feels ambiguous, I am happy to refine the wording in this section to more clearly emphasize the Foundation’s supportive role. Please feel free to share any additional thoughts!